astra/src/main/java/com/slack/astra/chunk/ChunkInfo.java

```diff
@@ -78,6 +79,9 @@ public static SnapshotMetadata toSnapshotMetadata(ChunkInfo chunkInfo, String ch
   // Path to S3 snapshot.
   private String snapshotPath;
 
+  // Size of chunk in bytes
+  private long sizeInBytes;
```

Comment:
Since this isn't a required param in the constructor, set a sensible default value (ie, `null`, `0`, `-1`) so that it's apparent if someone doesn't set it and then tries to read it what they should expect.